src/routes/commentRouter.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import {addComment, deleteComment, getComments} from '../controllers/commentController';
+import errorHandler  from '../middlewares/catchError';
+import isLoggedIn from '../middlewares/authenticate';
+import validate from '../middlewares/validate';
+import { validateComment } from '../validations/commentValidation';
+
+const commentRouter = express.Router();
+
+commentRouter.post(
+  '/trip/:tripId/comment',
+  validate(validateComment),
+  isLoggedIn,
+  errorHandler(addComment)
+);
+commentRouter.get(
+  '/trip/:tripId/comment',
+  errorHandler(getComments)
+);
+commentRouter.delete(
+  '/trip/comment/:commentId',
+  isLoggedIn,
+  errorHandler(deleteComment)
```

Comment:
You should add further authorization so that every user logged in could not be able to delete anyone's comment. I think this can belong to admin or any other manager.

Reply:
Please check the comment service file vs the comment controller file. only the owner of the comment can delete it. Kindly check the endpoint point

Reply:
@NNesta

Reply:
Ok I got it